tools/rostopic/src/rostopic/__init__.py

```diff
@@ -1602,7 +1602,7 @@ def _resource_name_package(name):
         return None
     return name[:name.find('/')]
 
-def create_publisher(topic_name, topic_type, latch):
+def create_publisher(topic_name, topic_type, latch, disable_rostime=True):
```

Comment:
It would be nice if you could add this new param to the docstring!

Reply:
yup, added in eeb9ce1

Reply:
there's a typo :laughing: (sorry)

Reply:
no problem! THX!